### PR TITLE
Use linter, move logic to page objects, support username overrides, support parallel tests

### PR DESCRIPTION
**Pull request by honzajavorek**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -35,6 +35,16 @@ Usage
     ...     dt = now_dt + timedelta(minutes=3)
     ...     session.tweet(dt, f'Second Tweet 💥 {dt.isoformat()}')
 
+Multiple Accounts
+-----------------
+
+If you have multiple accounts in your TweetDeck, you can specify the account to tweet from by ``session.tweet(dt, text, username='different_account')``. By default the tweet gets sent from the initial ``username``.
+
```
Comment (hroncok) [inline README.rst:42]: @honzajavorek thanks for clarification

Comment (honzajavorek) [reply README.rst:42]: It's a new feature 😄 d6b6be0

File: README.rst [hunk 2]
```diff
@@ -43,6 +53,8 @@ Obviously, TweetDeck authors can change anything anytime, which may or may not b
 .. image:: https://travis-ci.org/honzajavorek/schedule-tweet.svg?branch=master
     :target: https://travis-ci.org/honzajavorek/schedule-tweet
 
+The same tests run every time a Pull Request is created. Individual commit pushes are not tested to prevent collisions.
```
Comment (hroncok) [inline README.rst]: @honzajavorek IMHO pull request don't have access to encrypted Travis vars, or do they?

Comment (honzajavorek) [reply README.rst]: They have. But only mine. I didn't realize that. Basically, the current tests are only good as "nightly". It probably doesn't make sense to run them at any other times.

**Pull request merged**